README.md

```diff
@@ -6,27 +6,42 @@ We utilize puppeteer to get chrome performance traces.
 
 ## Testing on PC
 
-To run all you have to do is start the server and run the test suite on it.
+### Testing Sandstone components
+
+Start the server with Sandstone components and run the test suite on it.
 ```
-npm run test-all
+npm run test-all -- --theme=sandstone
 ```
 
 ```
-npm run serve
-npm run test
+npm run serve-sandstone

```

Comment:
Are there cases where `serve` and `test` must be used separately?
If that is not the case, you can remove the `serve` and `test` commands and leave only test-all, which accepts --target --theme --throttling as an option.

Reply:
`serve` is useful for developers in case of creating a new view for a component, for debugging purposes